packages/mui-material/src/locale/index.ts

```diff
@@ -103,23 +103,23 @@ export const arEG: Localization = {
   components: {
     MuiBreadcrumbs: {
       defaultProps: {
-        expandText: 'ظهر العنوان',
```

Comment:
A peek on the old translation before my changes, this current translation means in English: "the back (as in human back) of the title"....  my translation means is: "show Path".  This is why this PR most be merged. Cos current translation is totally wrong is some place.

Reply:
@flaviendelangle,  Could you please take a look at this PR, you have merged one PR for me in the past. see comments above. no one merged it yet.